chart/values.yaml

```diff
@@ -7,21 +7,21 @@ replicaCount: 1
 elasticsearch:
 # if `enabled = false` you need to set search.host
   enabled: true
-  imageTag: 6.8.6
+  imageTag: 6.8.8
   master:
     name: master
 
 postgresql:
   # if `enabled = false` you need to set database.host
   enabled: true
   image:
-    tag: 9.6.15
+    tag: 9.6
```

Comment:
Instead of this, we would probably be better off disabling this value altogether and just updating the version of the postgresql chart we use to latest.